gradle-plugins/src/main/kotlin/io.opentelemetry.instrumentation.muzzle-check.gradle.kts

```diff
@@ -34,11 +34,11 @@ val RANGE_COUNT_LIMIT = 10
 
 val muzzleConfig = extensions.create<MuzzleExtension>("muzzle")
 
-val muzzleTooling by configurations.creating {
+val muzzleTooling: Configuration by configurations.creating {
```

Comment:
We generally wouldn't specify the type for this sort of DSL usage, is it needed?